package-lock.json

```diff
@@ -15,35 +15,25 @@
         "connect-pg-simple": "^6.1.0",
         "cors": "^2.8.5",
         "d3": "^5.15.0",
-        "d3-selection-multi": "^1.0.1",
```

Comment:
lockfile is still version 2 -- latest npm creates version 3

Reply:
delete the file and create again using the latest npm

Reply:
Upgrading to version 3 increases the risk of compatibility. Version 2 has no impact on our project atm.

Reply:
what do you mean? what compatibility? the version is going to change with the pinboard pr anyway

Reply:
Changing your lock file to version 3 does not change your dev tools miraculously. Changing from NPM2 to 3 may introduce breaking changes or incompatibilities. Let's take it one step at a time. We just remove and updated some packages, let us evaluate the impact before changing the NPM version.